src/shell/autoexec.cpp

```diff
@@ -62,12 +62,12 @@ constexpr char char_cr = 0x0d; // carriage return
 // AUTOEXEC.BAT data - both source and binary
 // ***************************************************************************
 
-// Generated AUTOEXEC.BAT, un UTF-8 format
-static std::string autoexec_bat_utf8 = {};
-// Whether AUTOEXEC.BAT is already registered on the Z: drive
-static bool is_vfile_registered = false;
-// Code page used to generate Z:\AUTOEXEC.BAT from the internal UTF-8 version
-static uint16_t vfile_code_page = 0;
+// Generated AUTOEXEC.BAT, in UTF-8 format
+static std::optional<std::string> autoexec_bat_utf8 = {};
+// Generated AUTOEXEC.BAT, in DOS encoding, in a binary format,
+static std::vector<uint8_t> autoexec_bat_bin = {};
+// Code page used to generate 'autoexec_bat_bin'
+static std::optional<uint16_t> vfile_code_page = 0;
```

Comment:
`vfile_code_page = {}` is what you want to give a false `std::optional`. Otherwise it's a true optional with a value of 0.

Reply:
I forgot to change the init value. Fixed.